### ISSUE TITLE
subscribe to PR approvals

### ISSUE THREAD

**Issue by ghostsquad** [integrations/slack#987]
Is there anyway to show when a PR is approved?

**Comment by tebriel** [comment]
Yes, the new [Scheduled Reminders](https://github.blog/changelog/2019-11-14-scheduled-reminders-beta/) beta functionality does this. Sign up for the beta preview and it will be enabled for your organization as soon as we add more to the feature flag!

**Comment by don-hover** [comment]
I've read the documentation for the Scheduled Reminders:
https://hoverinc.atlassian.net/wiki/spaces/EPD/pages/2135883874/2021-04-12+Oauth+BE+Tech+Spec#GraphQL-response-format

I see a section that shows how to ignore Approved PRs, but I don't see anything specifically that shows how to be notified when a PR has been fully approved and is in a "Ready To Merge" state.

![Screen Shot 2021-04-16 at 4 38 10 PM](https://user-images.githubusercontent.com/77687833/115081472-53222700-9ed2-11eb-9064-ac3a84ae6d49.png)

Is there some part of this feature that could directly DM the PR author when their PR is approved/ready to merge?

**Comment by martha-truesource** [comment]
+1

I'd really like the option to be notified by Slack message when my PR is approved

**Comment by pgollucci** [comment]
👍

**Comment by martin056** [comment]
It would be really useful if it comes out of the box from the integration :+1:

**Comment by JCMais** [comment]
Is there any update on this feature?

**Comment by ido-hunters** [comment]
This feature will really help me as well :)

**Comment by jonbeacherdomuso** [comment]
Please implement this feature

**Comment by smacilwaine** [comment]
This would be a really great feature for my company as well. I spend so much time refreshing Github just to see if my PR is merged yet, and getting notified upon approval would be much more efficient.

**Comment by matiasf9** [comment]
Same here, I think it's as important as being notified when a new PR is opened.

**Comment by fsakalos** [comment]
+1

**Comment by prabhushrikant** [comment]
+1 for this feature. Would also like to know if new review is posted on your PR. Currently I only get notified when PR is created and merged:(

**Comment by danielpatricio-wa** [comment]
+1

**Comment by DannyC-BC** [comment]
+10000

**Comment by nandastone** [comment]
+1

**Comment by treborb** [comment]
+1

**Comment by jonnywyatt** [comment]
Please add this

**Comment by slapart** [comment]
+1

**Comment by michaelignat** [comment]
+ 1

**Comment by michaelignat** [comment]
+1

**Comment by amir-tastewise** [comment]
+1

**Comment by alishaevn** [comment]
checking in to see if this issue has been triaged at all? I don't see it mentioned in the current [road map](https://github.com/integrations/slack/#roadmap-and-upcoming-features) and we cannot do it ourselves as you all are [no longer accepting code contributions](https://github.com/integrations/slack/#questions-need-help).

thanks!

**Comment by mikemaring** [comment]
Still hoping to see this

**Comment by foamzou** [comment]
Need the feature as well

**Comment by noahtf13** [comment]
+1

**Comment by deweyredman** [comment]
+1

**Comment by muratdozen** [comment]
+1 definitely need the feature as well...

**Comment by dav-pascual** [comment]
+1

**Comment by kelvinsanchez15** [comment]
+1

**Comment by danieltharp** [comment]
3 years now that GitHub is being lapped on such a basic feature by Bitbucket, sheesh. You can't fire a message to slack on a PR approval?

**Comment by choladay** [comment]
+1

**Comment by aseerkt** [comment]
+1

**Comment by MilosMosovsky** [comment]
+1

**Comment by jacekendrick** [comment]
+1

**Comment by gentoomaniac** [comment]
+1
right now we have a slack workflow reacting on a :heavy_check_mark: on the PR message that we manually have to add 
:rofl:

**Comment by bernhardkaindl** [comment]
+1

**Comment by matt-theguyw1cat** [comment]
+1 its actually the only thing I really want from subs

**Comment by rutpshah** [comment]
@gentoomaniac
Can u share the workflow?

**Comment by Hojland** [comment]
+ 1

**Comment by VanekPetr** [comment]
+1

**Comment by maxence-flowie** [comment]
+100

**Comment by daniddelrio** [comment]
+1

**Comment by ayoukhananov** [comment]
+1

**Comment by tekeri** [comment]
+1

**Comment by rahul7668gupta** [comment]
+1
